### DISCUSSION TITLE
KubernetesPodOperator with GitSync

### DISCUSSION THREAD

**Discussion by DrChronoMartin** [apache/airflow#28175] [Q&A]
Is there any documentation or guide on how to volume mount GitSync DAG folder in KubernetesPodOperator assuming GitSync is setup using the official Helm Chart?

**Comment by potiuk** [comment]
I don't think so. But you can use `helm install --dry-run` and you can copy relevant resources from there to generate your pod_template_file. That's what I'd do at least
